nsqadmin/static/js/app_state.js

```diff
@@ -17,7 +17,11 @@ var AppState = Backbone.Model.extend({
         this.on('change:graph_interval', function(model, v) {
             localStorage.setItem('graph_interval', v);
         });
-        this.set('graph_interval', localStorage.getItem('graph_interval') || 'off');
+
+        var qp = _.object(_.compact(_.map(window.location.search.slice(1).split('&'),
+            function(item) { if (item) { return item.split('='); } })));
+        var interval = qp['t'] || localStorage.getItem('graph_interval') || 'off';
```

Comment:
also, while here, can we default to `2h` when you have graphite configured? (we used to do that see [here](https://github.com/nsqio/nsq/pull/323/files#diff-c0e97967285692126a91af4fdca200daL130) )